File: E-EstateV2/src/app/_interface/fieldConversion.ts
import { FieldStatus } from "./fieldStatus";

export interface FieldConversion {
    id: number
    otherCrop: string
    sinceYear: number | null
    fieldId: number
    createdBy: string
    createdDate: Date
    updatedBy: string
    updatedDate: Date
    fieldStatuses: FieldStatus[]
    fieldName: string
    isActive: boolean
    otherCropId:number

}

File: E-EstateV2 API/E-EstateV2 API/Models/FieldConversion.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace E_EstateV2_API.Models
{
    public class FieldConversion
    {
        //[Key]
        public int Id { get; set; }
        [ForeignKey("FieldId")]
        public int fieldId { get; set; }
        public Field Field { get; set; }
        public int otherCropId { get; set; }
        public int sinceYear { get; set; }
        public string createdBy { get; set; }
        public DateTime createdDate { get; set; }
        public string updatedBy { get; set; }
        public DateTime updatedDate { get; set; }
    }
}
